common/reviews/api/node-core-library.api.ts

```diff
@@ -1,3 +1,9 @@
+// @beta
+class Executable {
+  static spawnSync(command: string, args: string[], options?: IExecutableSpawnSyncOptions): child_process.SpawnSyncReturns<string>;
```

Comment:
>spawnSync [](start = 9, length = 9)

Add a note about adding another function that can take in an encoding and return a string.